qa/suites/fs/nfs/overrides/ignore_mgr_down.yaml

```diff
@@ -0,0 +1,7 @@
+# after some recent clog changes, MGR_DOWN warnings are generated before the
+# NFS test cases execution begins and this leads to the job getting failed.
```

Comment:
It's sufficient to say MGR_DOWN  is required because `mgr fail` will restart the mgr between tests.

Reply:
@dparmar18 I've tagged this with my testing branch. Please resolve this comment soon.

Reply:
> It's sufficient to say MGR_DOWN is required because `mgr fail` will restart the mgr between tests.

`mgr fail` happens before the tests are run, i haven't seen them between tests.

Reply:
```
2024-03-27T06:07:34.833 INFO:journalctl@ceph.mon.a.smithi184.stdout:Mar 27 06:07:34 smithi184 bash[21504]: audit 2024-03-27T06:07:34.221017+0000 mon.a (mon.0) 324 : audit [INF] from='client.? 172.21.15.184:0/2918680897' entity='client.admin' cmd='[{"prefix": "mgr fail", "who": "x"}]': finished
```

```
2024-03-27T06:07:52.025 INFO:tasks.cephfs_test_runner:Starting test: test_cephfs_export_update_at_non_dir_path (tasks.cephfs.test_nfs.TestNFS)
```

Reply:
> > It's sufficient to say MGR_DOWN is required because `mgr fail` will restart the mgr between tests.
> 
> `mgr fail` happens before the tests are run, i haven't seen them between tests.

Right, the `MgrTestCase.setup_mgrs` is called only when the test class is constructed. Anyway, please fix your comment.